### PR TITLE
Update Cyclone dds configuration

### PR DESCRIPTION
**Pull request by alsora**
This PR brings in corrections to how to configure cyclonedds.
Courtesy of @thijsmie

### CODE REVIEW

File: docs/setup/jetson.md [hunk 1]
```diff
@@ -118,7 +118,7 @@ The recommended way to run ROS2 on Jetson is to use pre-built Docker container i
 
 1. Set the default network interface by setting Cyclone DDS configuration.
 
-        export CYCLONEDDS_URI='<CycloneDDS><Domain><General><NetworkInterfaceAddress>l4tbr0</NetworkInterfaceAddress></General></Domain></CycloneDDS>'
+        export CYCLONEDDS_URI='<CycloneDDS><Domain><General><Interfaces><NetworkInterface name="l4tbr0" /></Interfaces></General></Domain></CycloneDDS>'
```
Comment (justinIRBT) [inline docs/setup/jetson.md:121]: Are the previous instructions outdated or not the correct way?  If the former, does this updated config work for our older releases?

Comment (thijsmie) [reply docs/setup/jetson.md:121]: Ah, I should have mentioned. The old style (`NetworkInterfaceAddress`) does still work, but the new style is available and recommended for ROS Humble and future releases.

Comment (thijsmie) [reply docs/setup/jetson.md:121]: No, as far as I can see Galactic is on the CycloneDDS 0.8.x release train so there the new style is not implemented.

Comment (justinIRBT) [reply docs/setup/jetson.md:121]: Should we break out separate instructions for galactic and humble where the configs differ between the two

Comment (thijsmie) [reply docs/setup/jetson.md:121]: That is probably a good idea then. Do you want (1) two separate files for humble/galactic or (2) mention the difference inline? Also, do you want me to change my contribution or restructure it yourself? I'm okay with either :slightly_smiling_face:

Comment (justinIRBT) [reply docs/setup/jetson.md:121]: @alsora What is your opinion?

Comment (alsora) [reply docs/setup/jetson.md:121]: I think that for now we should go with a disclaimer / inline comment.
Eventually we will fully support multiple versions for the docs and we can take care of it appropriately

**Comment by shamlian** [comment]
@thijsmie I'm sorry that this got neglected. Do you have any interest to revive it? If not, I can try to piece it back together on top of the current main.

**Comment by thijsmie** [comment]
No worries, I was just cleaning up repositories, I was not really intending to close this. I am no longer affiliated with the CycloneDDS project so maybe it is better if someone else handles it.

**Comment by shamlian** [comment]
Thanks; wishing you the best.

**Pull request closed without merging**